unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/SettingsPanelHUD/Scripts/ControlsModule/SettingsControlModel.cs

```diff
@@ -0,0 +1,40 @@
+using ReorderableList;
+using UnityEngine;
+
+namespace DCL.SettingsPanelHUD.Controls
+{
+    [System.Serializable]
+    public class SettingsControlGroupList : ReorderableArray<SettingsControlGroup>
+    {
+    }
+
+    [System.Serializable]
+    public class SettingsControlList : ReorderableArray<SettingsControlModel>
+    {
+    }
+
+    [System.Serializable]
+    public class SettingsControlGroup
+    {
+        [Reorderable]
+        public SettingsControlList controls;
+    }
+
+    [CreateAssetMenu(menuName = "Settings/Configuration/Controls/Default Control", fileName = "DefaultControlConfiguration")]
+    public class SettingsControlModel : ScriptableObject
+    {
+        public string title;
+        public SettingsControlView controlPrefab;
+        public SettingsControlController controlController;
+
+        public SettingsControlModel(
+            string title,
+            SettingsControlView controlPrefab,
+            SettingsControlController controlController)
+        {
+            this.title = title;
+            this.controlPrefab = controlPrefab;
+            this.controlController = controlController;
+        }
```

Comment:
Constructors can be used on `ScriptableObject` classes like this? In `Monobehaviour` classes this is heavily discouraged.

Reply:
Yes, you're completely right. I introduced these constructors (as default practice) when I created the class for the first time (when it was not still ScriptableObject) and I forgot to readapt it after work with SO. At least, these constructors are only be used in the unit tests and it is not affecting to the runtime logic... but I'm going to fix it anyway in several classes when it is happening, of course :)

Thanks! :D

Reply:
Fixed in this new PR (in order to avoid mix things between the part I and II): https://github.com/decentraland/explorer/pull/1784